lib/google_crawler_web/controllers/api/user_auth.ex

```diff
@@ -0,0 +1,35 @@
+defmodule GoogleCrawlerWeb.Api.UserAuth do
```

Comment:
Where we use this module? I can't find in this PR 🤔

Reply:
@hanam1ni This will be used in next PR, because there is no route for api yet. This will be used in `routes.rb`.

```
defmodule GoogleCrawlerWeb.Router do
  use GoogleCrawlerWeb, :router

  import GoogleCrawlerWeb.UserAuth
  import GoogleCrawlerWeb.Api.UserAuth

  scope "/api", GoogleCrawlerWeb, as: :api do
    pipe_through [:api, :authenticate_user_by_token]

    resources "/keywords", Api.KeywordController, only: [:show]
  end
end   
```